setup.cfg

```diff
@@ -25,9 +25,9 @@ include_package_data = True
 install_requires =
     ; By default, we only consider core dependencies required to use Ralph as a
     ; library (mostly models).
-    langcodes==3.3.0
-    pydantic[dotenv,email]==1.10.2
-    rfc3987==1.3.8
+    langcodes>=3.2.0
+    pydantic[dotenv,email]>=1.9.0
```

Comment:
Could we require `pydantic>=1.10`? It includes a lot of fixes and improvements (and also a fix for JSON types for hypothesis that we made)